### PR TITLE
Implemented Calculator Component

### PR DESCRIPTION
**Pull request by vangelif**
- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called components.
- Created a new Calculator.js file.
- Created a React component that matches the design:
- Developed just the UI of the calculator
- Created 1 child component
- Imported it and use it in your top-level component: App.js
- Tested it in the browser

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+    //     <a
+    //       className="App-link"
+    //       href="https://reactjs.org"
+    //       target="_blank"
+    //       rel="noopener noreferrer"
+    //     >
+    //       Learn React
+    //     </a>
+    //   </header>
+    // </div>
```
Comment (Gambit142) [inline src/App.js]: - [x] Kindly remove all commented-out codes in this file. Leaving such codes make your code look untidy and hard to read. Kindly remove them. 👍

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            </button>
+            <button type="button">.</button>
+            <button type="button" className="math-symbol">
+              =
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
```
Comment (Gambit142) [inline src/Components/Calculator.js]: - [x] One of the project requirements is to create at least 1 child component of the `Calculator` component. Kindly create a child component and pass props from the `Calculator` component to the child component. 👍 💻

File: src/index.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+// import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```
Comment (Gambit142) [inline src/index.js]: - [x] Kindly remove all commented-out codes in this file. Leaving such codes make your code look untidy and hard to read. Kindly remove them. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @vangelif 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Created `Calculator` component ✔️ 
- Created a `components` directory ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ShadyShawkat — APPROVED** [review]
## Hi @vangelif, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**